tests/examples/modes/on-blur.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { test, testWithoutJS, expect } from 'tests/setup/tests'
+
+const route = '/examples/modes/on-blur'
+
+test.only('With JS enabled', async ({ example }) => {
+  const { firstName, email, button, page } = example
+
+  await page.goto(route)
+
+  // Render
+  await example.expectField(firstName)
+  await example.expectField(email)
+  await expect(button).toBeEnabled()
+
+  // Client-side validation
+  await firstName.input.click()
+  await email.input.click()
+  await button.focus()
+
+  await example.expectError(
+    firstName,
+    'String must contain at least 1 character(s)',
+  )
+  await example.expectError(
+    email,
+    'String must contain at least 1 character(s)',
+  )
+
+  await button.click()
+  await expect(firstName.input).toBeFocused()
+
+  // Make first field be valid, focus goes to the second field
+  await firstName.input.fill('John')
+  await button.click()
+  await example.expectValid(firstName)
+  await expect(email.input).toBeFocused()
+
+  // Try another invalid message
+  await email.input.fill('john')
+  await example.expectError(email, 'Invalid email')
+
+  // Make form be valid
+  await email.input.fill('john@doe.com')
+  await example.expectValid(email)
+
+  // Submit form
+  button.click()
+  await expect(button).toBeDisabled()
+
+  const actionResult = await page
+    .locator('#action-data > pre:visible')
+    .innerText()
+
+  await expect(actionResult).toContain('"firstName": "John"')
+  await expect(actionResult).toContain('"email": "john@doe.com"')
+})
+
+testWithoutJS('With JS disabled', async ({ example }) => {
```

Comment:
Nothing different here compared to onSubmit, right?

Reply:
Right